tutorials/MD/setup.md

```diff
@@ -0,0 +1,34 @@
+# Molecular Dynamics (MD) Tutorial
```

Comment:
Why is this being added? It's going to override the files from #120?

If we want to add this information it should go in the README.md file.

Reply:
By mistake. Removed now.